docs/animations/CompositionAnimations.md

```diff
@@ -7,6 +7,9 @@ keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp
 
 # Composition Animations in XAML
 
+> [!WARNING]
+> These animation types have been updated with some breaking changes, please refer to the docs for the [`AnimationSet`](AnimationSet.md) and [`ImplicitAnimationSet`](ImplicitAnimationSet.md) types. For a comprehensive list of all the available APIs, use the .NET API browser instead.
+

```

Comment:
This is much more of a release notes thing. Are all the samples below up-to-date to work with the new version? If so, I don't think we need this note here.

Reply:
Not really, I've actually moved this whole page to the archive and split the docs into separate pages, since we now have the whole distinction between implicit/explicit animations (and the animation builder too) that wasn't there before. As in, this original docs page was covering mostly just the implicit ones in the old implementation, so I figured we'd just leave this here to avoid breaking links but then start fresh with the other new docs pages? 🤔